server/src/main/java/ro/lbi/sqliggybank/server/Resources/UserResource.java

```diff
@@ -232,7 +232,10 @@ public Response loginUser(String body) {
 			Account account = mapper.readValue(body, Account.class); // read object in Account class
 			User user = userDAO.findByUsername(account.getUsername()).orElseThrow(() -> new NotFoundException("No such username."));
 			if (!hasher.verifyHash(account.getPassword(), user.getPassword())) { // wrong password, eject client
-				return Response.status(Response.Status.FORBIDDEN).entity("Invalid username and password combination!").build();
+				return Response.
+						status(Response.Status.FORBIDDEN).
+						entity(new GenericResponse(Response.Status.FORBIDDEN.getStatusCode(), "Invalid username and password combination!"))
+						.build();
```

Comment:
The dots are supposed to be in front of every new method, so:
Response
.status( ... )
.entity( ... )
.build();

Is the right way of writing it.

Reply:
I'll fix it personally.